_translator-files/po/nl/Running-a-Server.po

```diff
@@ -882,10 +871,8 @@ msgstr "`sudo systemctl start jamulusTogglerec`"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:322
-#, fuzzy
-#| msgid "You can see the result of these commands if you run `systemctl status jamulus`, or by viewing the logs."
 msgid "You can see the result of these commands if you run `systemctl status jamulus-headless` (or the respective service name you specified manually). You can also view your (sys)log."
-msgstr "Je kunt het resultaat van deze commando's zien als je `systemctl status jamulus` uitvoert of door de logs te bekijken."
+msgstr "Je kunt het resultaat van deze commando's zien als je `systemctl status jamulus-headless`(of de betreffende servicenaam die u handmatig hebt opgegeven) uitvoert. Je kunt ook de (sys)log bekijken."
```

Comment:
@henkdegroot spacing?

Reply:
You've got eagle eyes, corrected on weblate